Sources/PackageCollections/PackageCollections.swift

```diff
@@ -454,16 +454,16 @@ public struct PackageCollections: PackageCollectionsProtocol {
     private func refreshCollectionFromSource(source: PackageCollectionsModel.CollectionSource,
                                              trustConfirmationProvider: ((PackageCollectionsModel.Collection, @escaping (Bool) -> Void) -> Void)?,
                                              callback: @escaping (Result<Model.Collection, Error>) -> Void) {
-        if let errors = source.validate()?.errors() {
```

Comment:
We have the same logic in the provider. Removing this so we don't have to duplicate storage removal logic (i.e., L464-466) here.